ArduSub/wscript

```diff
@@ -20,6 +20,7 @@ def build(bld):
             'AP_Arming',
             'AP_KDECAN',
             'AP_RCMapper',
+            'AP_IBus_Telem',
```

Comment:
We have a `Tools/ardupilotwaf/ardupilotwaf.py` for libraries common to all vehicles.  

But none of the other telemetry libraries are included in Sub (they don't do RC!), so we should probably just limit ourselves to adding it to Rover, Plane and Copter's `wscript` files.

Reply:
Currently the ibus telemetry is added in the AP_Vehicle, therefor it was added to `Tools/ardupilotwaf/ardupilotwaf.py`.
I guess I can remove it from the Rover/Plane/Copter/Sub wscript files?

If I would remove it from `Tools/ardupilotwaf/ardupilotwaf.py` and just put it in the wscript of Rover/Plan/Copter, will Sub still compile as it will need the ibus telem library (because it extends AP_Vehicle) right?

Reply:
Its definitely one or the other :-)

Suggest we leave it in `ardupilotwaf.py` and remove it from the `wscript`s